TS/docs/file_format_spec.txt

```diff
@@ -22,9 +22,9 @@ struct Image
 {
     char ident;   // marks start of image
     // Character value depends on image type, i.e. 'I' for PNG and 'J' for JPEG
+    double display_scale; // display scale of image
+    wxInt32 imagelen; // length of image (4 bytes)

```

Comment:
just say `int` like elsewhere in this file.. or `int64_t`.. best to keep this not wxwidgets specific where possible

Reply:
Ok, understood. This might for other parsers not being based on wxWidgets, of course.